docs/notebooks/tutorials/bootstrap_current.ipynb

```diff
@@ -0,0 +1,958 @@
+{
```

Comment:
<ul><li>rather than writing out all the zero modes might be clearer to use the <code>modes</code> argument.</li><li>also may want to explain why we use a non-symmetric current profile.</li><li>Could mention <code>Zeff</code> option as well</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>